README.md

```diff
@@ -1,73 +1,90 @@
 # ethstorage-sdk
-used for uploading and downloading blobs. The sdk is implemented based on [kzg-wasm](https://github.com/ethereumjs/kzg-wasm/), support the browser environment.
```

Comment:
I think a better way to approach this work is to propose a specification first. This way, the team can review it and provide feedback on what the SDK interface should look like. Otherwise, it might result in wasting time going back and forth several times.

Reply:
I also suggest referencing other SDKs for their design, such as the AWS S3 SDK or the IPFS JS SDK.